src/llm_models/anthropic.rs

```diff
@@ -80,7 +80,7 @@ impl LLMModel for AnthropicModels {
         json_schema: &Value,
         function_call: bool,
         max_tokens: &usize,
-        temperature: &u32,
+        temperature: &f32,
```

Comment:
This is never externally exposed so should be a safe change. We only set it with the constructor that hasn't changed.